Gopkg.lock

```diff
@@ -1,26 +1,6 @@
 # This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
 
 
-[[projects]]
-  digest = "1:7e1b8031787f1a7d7fb24b792d3d9c978d04f4f04d564c8925f1f613784b431b"
-  name = "bitbucket.org/atlassian/go-asap"
-  packages = [
-    ".",
-    "keyprovider",
-    "validator",
-  ]
-  pruneopts = "NT"
-  revision = "805ddb2a51bc1aa003b3e08603171f9658ef4e02"
-  version = "1.0.0"
-
-[[projects]]
-  digest = "1:d301ab1fc93c09892c70ae2ace85fa1680e3c8f3c0c5961cf9b5e12b64911f97"
-  name = "bitbucket.org/atlassianlabs/restclient"
```

Comment:
We need to opensource this client

Reply:
It waaaaaaaaaaas under https://github.com/scottgreenup/restclient :stuck_out_tongue_closed_eyes: